commit e53ace51d284f9b032c563ff53440c85d1cb9e12 (refs/changes/57/4657/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-09T10:50:20-07:00 (1 year, 2 months ago)
    
    TRITON-676 headnode setup fails in sdc-init.js after node-sdc-clients update from TRITON-593

diff --git a/scripts/sdc-deploy.js b/scripts/sdc-deploy.js
index a3028154..fc8453e1 100755
--- a/scripts/sdc-deploy.js
+++ b/scripts/sdc-deploy.js
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var async = require('/usr/node/node_modules/async');
@@ -52,6 +52,7 @@ function initSapiClient(cb) {
     sapi = new sdc.SAPI({ // intentionally global
         log: log,
         url: sapiUrl,
+        version: '~2',
         agent: false
     });
     return cb(null);
diff --git a/scripts/sdc-init.js b/scripts/sdc-init.js
index 81037658..1c388fbc 100755
--- a/scripts/sdc-init.js
+++ b/scripts/sdc-init.js
@@ -253,6 +253,7 @@ function initSapiClient(cb) {
     self.sapi = new sdc.SAPI({
         log: self.log,
         url: 'http://' + self.config.sapi_admin_ips,
+        version: '~2',
         agent: false
     });
     return cb(null);
